### PR TITLE
Remove dependencies pinned for stable release

### PR DESCRIPTION
**Pull request by NeonDaniel**
# Description
Update ovos dependencies to stable versions
Remove pinned versions
Update skills to latest alpha versions

# Issues

# Other Notes

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -5,8 +5,8 @@ ovos-plugin-common-play~=0.0.5
 
 neon-utils[network]~=1.8,>=1.8.2
 
-ovos-utils~=0.0.35
-ovos-bus-client~=0.0.5
+ovos-utils~=0.0.37
+ovos-bus-client~=0.0.7
```
Comment (JarbasAl) [inline requirements/requirements.txt:9]: recommend 0.0.8

File: requirements/requirements.txt [hunk 2]
```diff
@@ -5,8 +5,8 @@ ovos-plugin-common-play~=0.0.5
...
 neon-transformers~=0.2
 ovos-config~=0.0.11
 ovos-plugin-manager~=0.0.24
```
Comment (JarbasAl) [inline requirements/requirements.txt:12]: recommend 0.0.25

File: requirements/requirements.txt [hunk 3]
```diff
@@ -5,8 +5,8 @@ ovos-plugin-common-play~=0.0.5
 
 neon-utils[network]~=1.8,>=1.8.2
 
-ovos-utils~=0.0.35
-ovos-bus-client~=0.0.5
+ovos-utils~=0.0.37
+ovos-bus-client~=0.0.7
 neon-transformers~=0.2
 ovos-config~=0.0.11
```
Comment (JarbasAl) [inline requirements/requirements.txt:11]: recommend 0.0.12

File: requirements/requirements.txt [hunk 4]
```diff
@@ -28,16 +28,11 @@ neon-utterance-normalizer-plugin~=0.1
 # TODO: Patching test failures https://github.com/NeonGeckoCom/NeonCore/actions/runs/4888865255/jobs/8727164992?pr=422
 requests < 2.30.0
 
-# TODO: Pinning latest stable release
-ovos-workshop==0.0.12
```
Comment (JarbasAl) [inline requirements/requirements.txt:32]: recommend 0.0.15

Comment (NeonDaniel) [reply requirements/requirements.txt:32]: Not actually a dependency here, it will be updated in neon-utils though

**Comment by JarbasAl** [comment]
>  ImportError: cannot import name 'get_handler_name' from 'ovos_utils.messagebus' (/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/ovos_utils/messagebus.py)

seems autoformat deleted this backwards compat import since it was unused in that file, its ovos_utils.event

**Comment by NeonDaniel** [comment]
> > ImportError: cannot import name 'get_handler_name' from 'ovos_utils.messagebus' (/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/ovos_utils/messagebus.py)
> 
> seems autoformat deleted this backwards compat import since it was unused in that file, actual import is `ovos_utils.events`

Looks like this method is duplicated in `ovos_utils.__init__` and `ovos_utils.events`

**Pull request merged**